### PR TITLE
Update deps and minor fixes

### PR DESCRIPTION
**Pull request by bghgary**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -15,16 +15,19 @@ FetchContent_Declare(cmake-extensions
 FetchContent_Declare(ios-cmake
     GIT_REPOSITORY https://github.com/leetal/ios-cmake.git
     GIT_TAG 04d91f6675dabb3c97df346a32f6184b0a7ef845)
+FetchContent_Declare(arcana
+    GIT_REPOSITORY https://github.com/microsoft/arcana.cpp.git
+    GIT_TAG f2757396e80bc4169f2ddb938ce25367a98ffdd0)
```
Comment (docEdub) [inline CMakeLists.txt:20]: What will happen if the arcana Git tag listed here is different than the one listed in JsRuntimeHost? Will CMake complain? Or will it check out one on top of another?

File: CMakeLists.txt [hunk 2]
```diff
@@ -47,13 +50,13 @@ option(BABYLON_NATIVE_CORE_GRAPHICS "Include Babylon Native Core Graphics" ON)
...
 ## Plugins
 option(BABYLON_NATIVE_PLUGIN_EXTERNALTEXTURE "Include Babylon Native Plugin ExternalTexture." ON)
+option(BABYLON_NATIVE_PLUGIN_TESTUTILS "Include Babylon Native Plugin TestUtils." ON)
 option(BABYLON_NATIVE_PLUGIN_NATIVECAMERA "Include Babylon Native Plugin NativeCamera." ON)
 option(BABYLON_NATIVE_PLUGIN_NATIVECAPTURE "Include Babylon Native Plugin NativeCapture." ON)
 option(BABYLON_NATIVE_PLUGIN_NATIVEENGINE "Include Babylon Native Plugin NativeEngine." ON)
 option(BABYLON_NATIVE_PLUGIN_NATIVEINPUT "Include Babylon Native Plugin NativeInput." ON)
 option(BABYLON_NATIVE_PLUGIN_NATIVEOPTIMIZATIONS "Include Babylon Native Plugin NativeOptimizations." ON)
 option(BABYLON_NATIVE_PLUGIN_NATIVETRACING "Include Babylon Native Plugin NativeTracing." ON)
-option(BABYLON_NATIVE_PLUGIN_TESTUTILS "Include Babylon Native Plugin TestUtils." ON)
```
Comment (docEdub) [inline CMakeLists.txt:56]: Just curious why this got moved up.

File: CMakeLists.txt (hunk 1 above)
Comment (docEdub) [reply CMakeLists.txt:20]: CMake docs say the first one it sees wins.

**Review by docEdub — APPROVED** [review]

File: CMakeLists.txt (hunk 2 above)
Comment (bghgary) [reply CMakeLists.txt:56]: Alphabetical sort

**Pull request merged**